README.md

```diff
@@ -42,6 +46,13 @@ After generating the configure script with ``autogen``,  ``./configure --help``
 - ``make``
 - ``sudo make install``
 
+Run unit tests:
+
+- ``make check``
+
+Note that if you configured PointCloud using a non-standard LibGHT location, you may need to add its location to the ``LD_LIBRARY_PATH`` environment variable. For example:
```

Comment:
Yes, this is based on actual tests. And yes, it only occurs when using Autotools. CMake uses rpath or something and does not suffer the same problem.